### ISSUE TITLE
how to use many ardupilot_gazebo models ? 2 or 3 iris demos?

### ISSUE THREAD

**Issue by sofiathefirst** [khancyr/ardupilot_gazebo#35]
is there a solution to simulate many models in gazebo?

**Comment by linuxsocke** [comment]
Yes look here https://github.com/SwiftGust/ardupilot_gazebo#multi-vehicle-simulation

But be sure that you have mounted a gazebo imu and gps sensor.

**Comment by khancyr** [comment]
you can look here : https://github.com/Intelligent-Quads/iq_tutorials on how to do multidrone simulations !
